controllers/case-controller.js

```diff
@@ -26,16 +26,20 @@ const CaseController = Case => {
   };
 
   const listCases = async query => {
+    const limit = query.limit && parseInt(query.limit, 10);
+    const offset = query.offset && parseInt(query.offset, 10);
     const queryObject = {
       order: [['createdAt', 'DESC']],
       where: {
         helpline: query.helpline || '',
       },
+      limit,
+      offset,
     };
 
-    const cases = await Case.findAll(queryObject);
-    const withContactInfo = await Promise.all(
-      cases.map(async caseItem => {
+    const { count, rows } = await Case.findAndCountAll(queryObject);
```

Comment:
There's a `findAndCountAll`!  That's awesome.  Though it's good to keep in mind that behind the scenes, sequelize is probably making at least two queries.

Reply:
Yes, in fact it does, you can check that in the console after the function is called. Should I leave some comment around this?